### DISCUSSION TITLE
Updating runner target

### DISCUSSION THREAD

**Discussion by williamcrdv** [phil-opp/blog_os#1225] [Questions] [answered]
[target.'cfg(target_os = "none")']
runner = "bootimage runner"

The command used is targeting linux platforms. I need to be able to update this to add .exe. Where can I find the config for the runner?

**Comment by bjorn3** [comment]
Do you mean the `bootimage` command invoked here? If so, windows will add the `.exe` if necessary on it's own. Or do you mean the produced kernel? The produced kernel is not a windows executable in the PE format. Instead it is a baremetal program in the ELF format. This is independent of the system on which you build it. The bootloader crate expects an ELF file, so that is what is specified in the x86_64-blog_os.json target specification. Are you getting an error despite that? If so which one?

**Reply by williamcrdv** [reply]
I have the qemu location in my environment path but it still cannot find the executable.

**Reply by williamcrdv** [reply, marked as answer]
Ha, apparently I just needed to reopen my power shell. Thanks.
